src/stories/LineVisScales.stories.tsx

```diff
@@ -41,12 +58,12 @@ LogForX.args = {
   abscissaParams: { scaleType: ScaleType.Log },
 };
 
-export const SymLogForX = Template.bind({});
-SymLogForX.storyName = 'Sym Log for X';
-SymLogForX.args = {
+export const SqrtForX = Template.bind({});
+SqrtForX.storyName = 'Sqrt for X';
+SqrtForX.args = {
   dataArray: dataArrayForXLog,
   domain: domainForXLog,
-  scaleType: ScaleType.SymLog,
+  scaleType: ScaleType.Sqrt,
 };
```

Comment:
new stories for the square scale

Reply:
Not sure it is worth adding those as we will remove `Sqrt` from the scale available for axes ?